File: baekjoon/src/bruteForce/No1436.java
package bruteForce;

import java.io.*;
import java.util.StringTokenizer;

/**
 * https://www.acmicpc.net/problem/1436
 */
public class No1436 {
    public static final BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
    public static final BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
    public static StringTokenizer tokenizer;

    public static int getNumberOfTheEnd(int order) {
        int count = 0;
        int number = 666;
        while (count != order) {
            if (String.valueOf(number++).contains("666"))
                count++;
        }
        return number - 1;
    }

    public static void main(String[] args) throws IOException {
        int order = Integer.parseInt(reader.readLine());
        writer.write(String.valueOf(getNumberOfTheEnd(order)));
        writer.flush();
    }
}

File: baekjoon/src/silver5/No1436.kt
package silver5

import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

/**
 * https://www.acmicpc.net/problem/1436
 */
private val reader = BufferedReader(InputStreamReader(System.`in`))
private val writer = BufferedWriter(OutputStreamWriter(System.out))

fun main() {
    val index = reader.readLine().toInt()
    var count = 0
    var number666 = 666

    while (true) {
        if (number666.toString().contains("666")) {
            count++
            if (count == index) {
                break
            }
        }
        number666++
    }

    writer.write("$number666")
    writer.flush()
}